idd/Energy+.idd.in

```diff
@@ -64816,50 +65000,43 @@ WaterHeater:HeatPump,
        \object-list ZoneNames
        \note Used only if Inlet Air Configuration is ZoneAirOnly or ZoneAndOutdoorAir.
        \note Otherwise, leave field blank.
-  A14, \field Tank Object Type
```

Comment:
@mjwitte The motivation was that they are redundant since the object type field is on the object itself. I suppose I hadn't considered the possibility that EnergyPlus would support a user specifying a `WaterHeater:Mixed` and `WaterHeater:Stratified` object with the same name. Does EnergyPlus really need to support that?

@Myoldmopar Changing this will change the transition rules. I don't think any other changes noted on this pull request will.